projects/microphysics/argo/end_to_end.py

```diff
@@ -0,0 +1,222 @@
+"""Configuration class for submitting end to end experiments with argo
+"""
+from copy import deepcopy
+import yaml
+import subprocess
+import dataclasses
+import base64
+import os
+from fv3net.artifacts.resolve_url import resolve_url
+import pathlib
+import sys
+
+
+WORKFLOW_FILE = pathlib.Path(__file__).parent / "argo.yaml"
+
+
+def load_yaml(path):
+    return yaml.safe_load(pathlib.Path(path).read_bytes())
+
+
+def submit(job, labels, other_parameters=None):
```

Comment:
Since this is a main public function, add a brief docstring and maybe a type for `job` so it's easy to see what goes in.  You describe it in the examples below, but this will make it abundantly clear.